paasta_tools/cli/cmds/secret.py

```diff
@@ -105,6 +105,16 @@ def add_subparser(subparsers):
         default=False,
         help="Optionally pass the plaintext from stdin",
     )
+    secret_parser.add_argument(
+        "--cross_environment",
+        required=False,
+        type=str,
+        help=(
+            "Provide motivation in case the same value is being duplicated "
```

Comment:
argument name doesn't correspond to help message, maybe rename it to `--cross-env-motivation`?

Reply:
done